app/src/main/java/helium314/keyboard/compat/EditorInfoCompatUtils.kt

```diff
@@ -31,13 +31,13 @@ object EditorInfoCompatUtils {
     @JvmStatic
     fun getHintLocales(editorInfo: EditorInfo?): List<Locale>? {
         if (editorInfo == null || Build.VERSION.SDK_INT < Build.VERSION_CODES.N) {
-            return null
+            return listOf()
```

Comment:
Practically irrelevant nitpick: I'd recommend to use `emptyList()` in such cases, as it doesn't create a new list.